cmd/kpod/common.go

```diff
@@ -133,3 +133,320 @@ func validateFlags(c *cli.Context, flags []cli.Flag) error {
+	},
+	cli.StringSliceFlag{
+		Name:  "device-write-bps",
+		Usage: "Limit write rate (bytes per second) to a device (e.g. --device-write-bps=/dev/sda:1mb)",
+	},
+	cli.StringSliceFlag{
+		Name:  "device-write-iops",
+		Usage: "Limit write rate (IO per second) to a device (e.g. --device-write-iops=/dev/sda:1000)",
+	},
+	cli.StringSliceFlag{
+		Name:  "dns",
+		Usage: "Set custom DNS servers",
+	},
+	cli.StringSliceFlag{
+		Name:  "dns-opt",
+		Usage: "Set custom DNS options",
+	},
+	cli.StringSliceFlag{
+		Name:  "dns-search",
+		Usage: "Set custom DNS search domains",
+	},
+	cli.StringFlag{
+		Name:  "entrypoint",
+		Usage: "Overwrite the default ENTRYPOINT of the image",
+	},
+	cli.StringSliceFlag{
+		Name:  "env, e",
+		Usage: "Set environment variables in container",
+	},
+	cli.StringSliceFlag{
+		Name:  "env-file", //
+		Usage: "Read in a file of environment variables",
+	},
+	cli.StringSliceFlag{
+		Name:  "expose",
+		Usage: "Expose a port or a range of ports (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "group-add",
+		Usage: "Add additional groups to join (default [])",
+	},
+	cli.StringFlag{
+		Name:  "hostname",
+		Usage: "Set container hostname",
+	},
+	cli.BoolFlag{
+		Name:  "interactive, i",
+		Usage: "Keep STDIN open even if not attached",
+	},
+	cli.StringFlag{
+		Name:  "ip",
+		Usage: "Container IPv4 address (e.g. 172.23.0.9)",
+	},
+	cli.StringFlag{
+		Name:  "ip6",
+		Usage: "Container IPv6 address (e.g. 2001:db8::1b99)",
+	},
+	cli.StringFlag{ //
+		Name:  "ipc",
+		Usage: "IPC Namespace to use",
+	},
+	cli.StringFlag{
+		Name:  "kernel-memory",
+		Usage: "Kernel memory limit (format: `<number>[<unit>]`, where unit = b, k, m or g)",
+	},
+	cli.StringSliceFlag{
+		Name:  "label",
+		Usage: "Set metadata on container (default [])",
+	},
+	cli.StringSliceFlag{ //
+		Name:  "label-file",
+		Usage: "Read in a line delimited file of labels (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "link-local-ip",
+		Usage: "Container IPv4/IPv6 link-local addresses (default [])",
+	},
+	cli.StringFlag{
+		Name:  "log-driver",
+		Usage: "Logging driver for the container",
+	},
+	cli.StringSliceFlag{
+		Name:  "log-opt",
+		Usage: "Logging driver options (default [])",
+	},
+	cli.StringFlag{
+		Name:  "mac-address",
+		Usage: "Container MAC address (e.g. 92:d0:c6:0a:29:33)",
+	},
+	cli.StringFlag{
+		Name:  "memory, m",
+		Usage: "Memory limit (format: <number>[<unit>], where unit = b, k, m or g)",
+	},
+	cli.StringFlag{
+		Name:  "memory-reservation",
+		Usage: "Memory soft limit (format: <number>[<unit>], where unit = b, k, m or g)",
+	},
+	cli.StringFlag{
+		Name:  "memory-swap",
+		Usage: "Swap limit equal to memory plus swap: '-1' to enable unlimited swap",
+	},
+	cli.Int64Flag{
+		Name:  "memory-swappiness",
+		Usage: "Tune container memory swappiness (0 to 100) (default -1)",
+	},
+	cli.StringFlag{
+		Name:  "name",
+		Usage: "Assign a name to the container",
+	},
+	cli.StringFlag{
+		Name:  "net",
```

Comment:
We've network-alias below.  This really feels like it should be network-namespace to me unless it's already established by kubernetes.

Reply:
We are following the Docker cli, so we can add netns, but need to have net as well.


Name: "netns, net"

I think works.